src/core/jsproxy.c

```diff
@@ -846,7 +845,6 @@ JsMethod_Vectorcall(PyObject* self,
 
   // Recursion error?
   FAIL_IF_NONZERO(Py_EnterRecursiveCall(" in JsProxy_Vectorcall"));
-  bool success = false;
```

Comment:
Here it was used uninitialized in the finally block if the `FAIL_IF_NONZERO` above failed.

Reply:
It's weird though: all the following declarations are ALSO used uninitialized when there's a recursion error, but somehow `-Werror=sometimes-uninitialized` only caught the first one?

Reply:
Funny thing though, the following declarations are also used uninitialized in that case but the compiler only warns about the first...